rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc

```diff
@@ -2,6 +2,32 @@
 
 RPA Manager is the cloud-based control plane for your automation projects.
 
+[[april-20-2023]]
+== April 20, 2023
+
+// == Fixed Issues
+include::partial$release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+2]
+
+[%header,cols="6a,1a"]
+|===
+|Issue Resolution |ID
+| RPA Bots no longer remain blocked after failing to create a secure session because RPA Manager now pauses a failing run configuration after 5 consecutive errors. | W-12718095
```

Comment:
```suggestion
| RPA Bots no longer remain blocked after failing to create a secure session because RPA Manager now pauses a failing run configuration after five consecutive errors. | W-12718095
```